dev_guide/application_memory_sizing.adoc

```diff
@@ -1,5 +1,5 @@
 [[dev-guide-application-memory-sizing]]
-= Application memory sizing
+= Application memory and ephemeral storage sizing
```

Comment:
rather than coupling this with the memory document, can we create a separate dev-guide document specific to ephemeral storage sizing?

Reply:
OK